<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Lenovo" name="com.TestNg_Concepts.Timeout_Test" tests="1" failures="0" timestamp="2021-03-06T10:47:59 IST" time="2.138" errors="1">
  <testcase name="navigateToPage" time="2.138" classname="com.TestNg_Concepts.Timeout_Test">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.TestNg_Concepts.Timeout_Test.navigateToPage() didn&#039;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.TestNg_Concepts.Timeout_Test.navigateToPage() didn't finish within the time-out 2000
at java.lang.Thread.interrupt(Unknown Source)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:60)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:41)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- navigateToPage -->
</testsuite> <!-- com.TestNg_Concepts.Timeout_Test -->
